packages/compass-serverstats/src/components/index.jsx

```diff
@@ -1,21 +1,35 @@
 require('./index.less');
 
 const React = require('react');
+const { WorkspaceContainer, css, spacing } = require('@mongodb-js/compass-components');
+const { StatusRow } = require('hadron-react-components');
 
 const GraphsComponent = require('./server-stats-graphs-component');
 const { realTimeDispatcher } = require('../d3');
 const ListsComponent = require('./server-stats-lists-component');
 const DBErrorComponent = require('./dberror-component');
-const TimeAndPauseButton = require('./time-and-pause-button');
 const DBErrorStore = require('../stores/dberror-store');
 const ServerStatsStore = require('../stores/server-stats-graphs-store');
-const { StatusRow } = require('hadron-react-components');
+const { ServerStatsToolbar } = require('./server-stats-toolbar');
 
 /**
  * The default interval.
  */
 const INTERVAL = 1000;
 
+const workspaceContainerStyles = css({
+  background: '#3D4247'
```

Comment:
Do we have this value somewhere in constants? And if not maybe we could introduce one?

Reply:
I couldn't find this value used anywhere else in Compass. Chatting with Claudia and looking through mocks the styles (and background color) here are soon to change so I'm thinking having this one is probably fine. I'll make it a variable in this file though 👍